### PR TITLE
Increase max input values for capacity sliders

### PR DESCRIPTION
**Pull request by KoenvanB**
This PR increases the maximum input value for several capacity sliders. The current values do not suffice e.g. in the case of provinces where one province hosts the production capacity for the entire country. Also, some max values are based on restrictions in the present (e.g. rooftop surface area available) which do not scale well with related changes in the future.

This PR makes the following increases to slider max values:

- rooftop solar pv capacities to 8x maximum roof surface area;
- nuclear plant, SMR plants, refinery p2h and flex mv batteries to at least 10000 MW;
- p2h heatpump by factor of 6;
- paper industry p2h to at least 25 MW;
- hydrogen salt cavern storage to at least 1000 TWh.

### CODE REVIEW

File: inputs/demand/industry/capacity_of_industry_other_paper_flexibility_p2h_electricity.ad [hunk 1]
```diff
@@ -15,7 +15,7 @@
       )
     )
 - priority = 0
-- max_value_gql = present:MAX(1.0,PRODUCT(DIVIDE(V(industry_useful_demand_for_other_paper_useable_heat,demand),(8760 * MJ_PER_MWH)),2.0))
+- max_value_gql = present:MAX(25.0,PRODUCT(DIVIDE(V(industry_useful_demand_for_other_paper_useable_heat,demand),(8760 * MJ_PER_MWH)),2.0))
```
Comment (mabijkerk) [inline inputs/demand/industry/capacity_of_industry_other_paper_flexibility_p2h_electricity.ad:18]: Why not use the same max value as for industry_chemical_refineries? If possible, I would use uniform max_values for uniform sliders.

File: inputs/flexibility/energy/capacity_of_energy_heat_flexibility_p2h_heatpump_electricity.ad [hunk 2]
```diff
@@ -12,7 +12,7 @@
       )
     )
 - priority = 0
-- max_value_gql = present:DIVIDE(DIVIDE(MAX(0.5 * Q(heat_production_total) * BILLIONS, V(energy_heat_flexibility_p2h_heatpump_ht_electricity, output_of_steam_hot_water)*2), V(energy_heat_flexibility_p2h_heatpump_ht_electricity,full_load_hours)), MJ_PER_MWH)
+- max_value_gql = present:DIVIDE(DIVIDE(MAX(3 * Q(heat_production_total) * BILLIONS, V(energy_heat_flexibility_p2h_heatpump_ht_electricity, output_of_steam_hot_water)*12), V(energy_heat_flexibility_p2h_heatpump_ht_electricity,full_load_hours)), MJ_PER_MWH)
```
Comment (mabijkerk) [inline inputs/flexibility/energy/capacity_of_energy_heat_flexibility_p2h_heatpump_electricity.ad:15]: Same comment as above, but then for the capacity_of_energy_heat_flexibility_p2h_boiler_electricity.

File: inputs/demand/industry/capacity_of_industry_other_paper_flexibility_p2h_electricity.ad (hunk 1 above)
Comment (KoenvanB) [reply inputs/demand/industry/capacity_of_industry_other_paper_flexibility_p2h_electricity.ad]: I have updated all industry flex p2h inputs to be at least 10000 MW, in line with the proposed change for industry_chemical_refineries.

File: inputs/flexibility/energy/capacity_of_energy_heat_flexibility_p2h_heatpump_electricity.ad (hunk 2 above)
Comment (KoenvanB) [reply inputs/flexibility/energy/capacity_of_energy_heat_flexibility_p2h_heatpump_electricity.ad:15]: I have updated `capacity_of_energy_heat_flexibility_p2h_boiler_electricity` by a factor of 6 as well.

File: inputs/supply/electricity/nuclear_plants/capacity_of_energy_power_nuclear_gen2_uranium_oxide.ad [hunk 3]
```diff
@@ -11,7 +11,7 @@
       		V(energy_power_nuclear_gen2_uranium_oxide, production_based_on_number_of_units)
     ))
 - priority = 0
-- max_value_gql = present:2*DIVIDE(DIVIDE(Q(total_electricity_consumed), V(energy_power_nuclear_gen2_uranium_oxide,full_load_hours)), MJ_PER_MWH)
+- max_value_gql = present:MAX(10000.0,2*DIVIDE(DIVIDE(Q(total_electricity_consumed), V(energy_power_nuclear_gen2_uranium_oxide,full_load_hours)), MJ_PER_MWH))
```
[the remark this answers is not in the document]
Comment (KoenvanB) [reply inputs/supply/electricity/nuclear_plants/capacity_of_energy_power_nuclear_gen2_uranium_oxide.ad:14]: I have updated all coal, gas, oil and the nuclear SMR plants to have the same 10000 max input value.

File: inputs/supply/hydrogen/hydrogen_production/capacity_of_energy_hydrogen_steam_methane_reformer_dispatchable.ad [hunk 4]
```diff
@@ -13,7 +13,7 @@
     )
 
 - priority = 0
-- max_value_gql = present:MAX(500,DIVIDE(Q(total_gas_consumed),PRODUCT(V(energy_hydrogen_steam_methane_reformer_dispatchable,full_load_hours),MJ_PER_MWH)))
+- max_value_gql = present:MAX(10000,DIVIDE(Q(total_gas_consumed),PRODUCT(V(energy_hydrogen_steam_methane_reformer_dispatchable,full_load_hours),MJ_PER_MWH)))
```
[the remark this answers is not in the document]
Comment (KoenvanB) [reply inputs/supply/hydrogen/hydrogen_production/capacity_of_energy_hydrogen_steam_methane_reformer_dispatchable.ad:16]: I have updated all hydrogen production plants (excluding wind/solar) to be at least 10000 MW.

**Review by mabijkerk — APPROVED** [review]
LGTM

**Pull request merged**